### DISCUSSION TITLE
Rename "scene item" to "source"?

### DISCUSSION THREAD

**Discussion by ChrisChinchilla** [WarmUpTill/SceneSwitcher#669] [Ideas]
Hey there, fantastic plugin, it's saving me lots of forgetting to do this during streams by just automating them :)

Though I was trying to toggle some sources' visibility and couldn't seem to figure out how to… Then I noticed what I actually wanted was "scene items"… Would it make any sense to rename that? Or did I misunderstand some OBS terminology myself?

**Comment by WarmUpTill** [comment]
Hi,
I assume you are referring to adjusting the visibility of an _instance_ of a source on a particular scene using the eye icon.
These are called "scene items" OBS internally (on the API level) so I just stuck with that name.

I get your point that this name can be a bit confusing for new users, but I fear that changing the name now would possibly confuse _existing_ users.

Sorry about the delayed response - I somehow missed this thread.

**Reply by ChrisChinchilla** [reply]
@WarmUpTill Ahhh, OK, no worries. Thanks for the great plugin!
